cmd/completion.go

```diff
@@ -18,14 +18,14 @@ var completionCmd = &cobra.Command{
 	Long: `To load completion run
 
 For zsh:
-source <(function completion zsh)
+source <(func completion zsh)
```

Comment:
that's an issue ;-) (function needs to stay here)
```suggestion
source <(function completion zsh)
```

Reply:
why?

Reply:
sorry overlooked that this is not the code generated. forget that ;)